src/app.rb

```diff
@@ -0,0 +1,85 @@
+require './util/json_storage'
+require_relative './modules/game_module'
+require_relative './modules/author_module'
+require_relative './modules/label_module'
+require_relative './modules/book_module'
+require_relative './functions/find_music_albums'
+require_relative './functions/find_genres'
+require_relative './functions/add_a_music_album'
+
+class App
+  include GameModule
+  include AuthorModule
+  include LabelModule
+  include BookModule
+
+  def initialize
+    @games = JSONStorage.load_data('games')
+    @authors = JSONStorage.load_data('authors')
+    @labels = JSONStorage.load_data('labels')
+    @books = JSONStorage.load_data('books')
+    @music_albums = JSONStorage.load_data('music_albums').empty? ? [] : JSONStorage.load_data('music_albums')
+    @genres = JSONStorage.load_data('genres').empty? ? [] : JSONStorage.load_data('genres')
+  end
+
+  ACTIONS = {
+    1 => :find_books,
+    2 => :find_music_albums,
+    3 => :find_movies,
+    4 => :find_games,
+    5 => :find_genres,
+    6 => :find_labels,
```

Comment:
- [ ] Kindly ensure that I can list labels after creating the Games and Labels 👍

Reply:
Fixed. Thanks